features/fixtures/generic/Plugins/Tweaks/Source/Tweaks/Tweaks_UPL.xml

```diff
@@ -21,14 +21,14 @@
 
     <gameActivityClassAdditions>
         <insert>
-            private void clearBugsnagCache() {
+            public void clearBugsnagCache() {
```

Comment:
What is this change for?

Reply:
ProGuard seems to hide private methods, making them unaccessible from JNI.

Reply:
Hmm. So this is simpler than adding it to the app's proguard rules. Ok :+1:

Reply:
Yes it seemed simpler to make these public than adding more proguard rules 🙃